src/main/resources/api.yaml

```diff
@@ -396,6 +518,38 @@ paths:
               schema:
                 $ref: '#/components/schemas/SprintSummaryDto'
 
+  /projects/{projectId}/sprints/{sprintId}/jira/sprints:
```

Comment:
Co do tego, to nie jestem pewna, czy to ma sens :( Nie umiem na razie odnaleźć Trzech Liczb z Jiry w API i nie wiem, czy identyfikacja sprintu nam w czymś pomaga :(